operator/pkg/model/translation/ingress/dedicated_ingress_test.go

```diff
@@ -166,3 +167,60 @@ func Test_getEndpointForIngress(t *testing.T) {
 		},
 	}, res)
 }
+
+func Test_translator_Translate(t *testing.T) {
+	type args struct {
+		m            *model.Model
+		enforceHTTPs bool
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    *ciliumv2.CiliumEnvoyConfig
+		wantErr bool
+	}{
+		{
+			name: "Conformance/DefaultBackend",
+			args: args{
+				m: &model.Model{
+					HTTP: defaultBackendListeners,
+				},
+				enforceHTTPs: true,
+			},
+			want: defaultBackendListenersCiliumEnvoyConfig,
+		},
+		{
+			name: "Conformance/HostRules",
+			args: args{
+				m: &model.Model{
+					HTTP: hostRulesListeners,
+				},
+				enforceHTTPs: true,
+			},
+			want: hostRulesListenersCiliumEnvoyConfig,
+		},
+		{
+			name: "Conformance/PathRules",
+			args: args{
+				m: &model.Model{
+					HTTP: pathRulesListeners,
+				},
+				enforceHTTPs: true,
```

Comment:
is the case with `enforceHTTPs=false` covered?

Reply:
yup, I am planning add more tests to cover such non-default installation.